src/DynamoCore/Search/SearchElements/ZeroTouchSearchElement.cs

```diff
@@ -136,4 +136,51 @@ private string GetIconName()
             }
         }
     }
+
+    internal class ReflectionZeroTouchSearchElement : NodeSearchElement
```

Comment:
I think this deserves a summary.

Reply:
is there a reason this derives from `NodeSearchElement` and not `ZeroTouchNodeSearchElement` to avoid duplicating some code?

Reply:
After looking further into this there isnt really a need for the `ReflectionZeroTouchSearchElement` so removed that for now, its fine to simply use the `ZeroTouchNodeSearchElement`